cypress/e2e/shared/explorerVisualizations.test.ts

```diff
@@ -130,7 +133,7 @@ describe('visualizations', () => {
 
           // cycle through all the visualizations of the data
           VIS_TYPES.forEach(type => {
-            if (type !== 'mosaic' && type !== 'band') {
+            if (type !== 'mosaic') {
```

Comment:
```suggestion
            if (type !== 'mosaic' && type !== 'gauge') {
```

^ at least on tools notebooks :(

Reply:
This e2e test currently does not fail with 'gauge'. Since this PR is limited to Band, let's address Gauge separately.